src/Sylius/Bundle/ProductBundle/Behat/ProductContext.php

```diff
@@ -289,4 +294,40 @@ public function theFollowingOptionTranslationsExist(TableNode $table)
 
         $manager->flush();
     }
+
+    /**
+     * @param ProductInterface $product
+     * @param array            $data
+     */
+    private function configureProductPricingCalculator(ProductInterface $product, array $data)
+    {
+        $product->getMasterVariant()->setPricingCalculator($data['pricing calculator']);
+
+        if (!isset($data['calculator configuration']) || '' === $data['calculator configuration']) {
```

Comment:
Yeah, but ``$data['caluculator configuration`]`` is not *de facto* an array - it's a string, that is parsed to array in ``$this->getPricingCalculatorConfiguration($data)`` method ;)